src/components/Home/ObjectList.jsx

```diff
@@ -11,7 +11,7 @@ const ObjectList = ({ title, seeMore, items }) => {
         </p>
         <Link
           to={seeMore}
-          className="text-2xl tracking-tighter font-normal text-gray-600"
+          className="text-2xl tracking-tighter font-normal text-gray-600 underline"
```

Comment:
Please add a hover state, maybe try to remove the underline on hover or increase the font size a little to be in tone with the buttons on the slider, for example

`hover:no-underline hover-scale--min`

Reference: https://github.com/Azordev/art-marketplace/blob/2ec9880440156c07df5dd33fd6648d8241526a63/src/components/Slider.js#L90